critiquebrainz/frontend/forms/review.py

```diff
@@ -42,7 +62,7 @@ class ReviewEditForm(Form):
             ('CC BY-NC-SA 3.0', lazy_gettext('Do not allow commercial use of this review, unless approved by MetaBrainz Foundation (<a href="https://creativecommons.org/licenses/by-nc-sa/3.0/" target="_blank">CC BY-NC-SA 3.0 license</a>)')),  # noqa: E501
         ],
         validators=[validators.DataRequired(message=lazy_gettext("You need to choose a license!"))])
-    language = SelectField(lazy_gettext("You need to accept the license agreement!"), choices=languages)
+    language = OptgroupSelectField(lazy_gettext("You need to accept the license agreement!"), choices=LANGUAGES)
```

Comment:
This would be then:
```
language = OptgroupSelectField(lazy_gettext("You need to accept the license agreement!"), choices=get_language_choices())
```